src/deluge/gui/views/automation_instrument_clip_view.h

```diff
@@ -160,8 +162,6 @@ class AutomationInstrumentClipView final : public ClipView, public InstrumentCli
 	                         int32_t firstPadY, int32_t secondPadX, int32_t secondPadY, bool modEncoderAction = false);
 
 	int32_t calculateKnobPosForModEncoderTurn(int32_t knobPos, int32_t offset);
-	bool isOnParameterGridMenuView();
```

Comment:
renamed isOnParameterGridMenuView() to is isOnAutomationOverview() and made the function public as its used by view.cpp